app/aws_meter.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Is there any reason to add this to the app layer? Would it makes sense to move it into a service and pass the pointer to the store during the `newAWSMeterService` call? The rest of the code looks like not related to the app at all, and I don't see any dependency.

Reply:
I would move it to `services/awsmeter` and rename the `newAWSMeterService` to just `New` method receiving the `store`.

Reply:
Thanks